### PR TITLE
ALS-1136: Upgrade handlebars, bootstrap, and httpd versions

### PR DESCRIPTION
**Pull request by ramari16**

### CODE REVIEW

File: pic-sure-hpds-ui/httpd/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM httpd:2.4.27-alpine
+FROM httpd:2.4.46-alpine
```
Comment (ramari16) [inline pic-sure-hpds-ui/httpd/Dockerfile:1]: This also needs to be changed in the project-specific Dockerfile

File: pic-sure-hpds-ui/httpd/httpd-vhosts.conf [hunk 2]
```diff
@@ -1,4 +1,3 @@
-Listen 0.0.0.0:80
```
Comment (ramari16) [inline pic-sure-hpds-ui/httpd/httpd-vhosts.conf:1]: This listen is redundant and conflicts with the one in httpd.conf in the latest version

File: pic-sure-hpds-ui/src/main/webapp/picsureui/index.html [hunk 3]
```diff
@@ -1,8 +1,6 @@
 <html>
 <head>
     <script src="/picsureui/webjars/requirejs/2.3.5/require.min.js" data-main="/picsureui/main"></script>
-    <link href="/picsureui/webjars/bootstrap/3.3.7-1/css/bootstrap.min.css"
-          rel="stylesheet">
```
Comment (ramari16) [inline pic-sure-hpds-ui/src/main/webapp/picsureui/index.html:5]: This is redundant, bootstrap is loaded via main.js

**Comment by chunacatsunflower** [comment]
This looks good.  I will merge this update after we finalize the code for our BDC release.

**Pull request merged**